RHOAIENG-16936 - add note (#587)
opendatahub-io/opendatahub-documentation: modules/installing-trustyai-service.adoc

[NOTE]
====
Install only one instance of the TrustyAI service in a project. Multiple instances in the same project can result in unexpected behavior.

> Installing TrustyAI into a namespace where non-OVMS models are deployed is not recommended, as it will cause errors in the TrustyAI service.

Reviewer: Slightly unclear on the "it" that will cause errors. Could we simplify to something like:  
```suggestion
Installing TrustyAI into a namespace where non-OVMS models are deployed is not recommended and can cause errors in the TrustyAI service.
```
or even remove the "is not recommended" for further simplicity 😄
```suggestion
Installing TrustyAI into a namespace where non-OVMS models are deployed can cause errors in the TrustyAI service.
```

Small note from IBM style guide, use _because_ instead of _as_ for sentences like these (barring our rewrite 😄): https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a
